src/Raven.Studio/typescript/models/database/tasks/ongoingTaskOlapEtlTransformationModel.ts

```diff
@@ -63,7 +65,7 @@ class ongoingTaskOlapEtlTransformationModel {
             ApplyToAllDocuments: this.applyScriptForAllCollections(),
             Collections: this.applyScriptForAllCollections() ? null : this.transformScriptCollections(),
             Disabled: false,
-            Name: this.name(),
+            Name: this.name().trim(),
```

Comment:
do we want that? usually we don't trim props.

Reply:
if someone save task outside studio with space then we effectively edit name by just saving in studio

Reply:
removed trim